### ISSUE TITLE
Just want to say thank you!

### ISSUE THREAD

**Issue by vannessqn** [Saramagrean/TKD_KWM-HD-Texture-Pack#5]
merry christmas and happy new year!!

**Comment by Saramagrean** [comment]
Yeah, should thank you HoaiTrung-san for hard work. I just edit cutscenes & made texture pack :)

**Comment by IrfanH495** [comment]
> Yeah, should thank you HoaiTrung-san for hard work. I just edit cutscenes & made texture pack :)

sorry if my comment is not in the right place, I want to ask.? 
Are you the one who made the HD texture pack for Wariors Orochi 3, and can I request a texture font like this
![041ef00014d955f8beafed1b](https://user-images.githubusercontent.com/78854859/218578114-ba19c0db-d7bd-4a36-8369-0376720b1f06.png)
![041ef000436d254cbeafed1b](https://user-images.githubusercontent.com/78854859/218578121-3bcf988b-6a3e-4d21-988f-81e5e026f713.png)
![041ef00093e65e58beafed1b](https://user-images.githubusercontent.com/78854859/218578126-1a8943b3-1eec-4979-b366-654afd3b3b78.png)
![041ef000c8a7a294beafed1b](https://user-images.githubusercontent.com/78854859/218578133-fd7cb605-aa23-400a-84aa-a19d9143eabe.png)
i want to download it on the ppsspp forum but, the size is too big for me, i only need 4 pictures can you share it.

**Comment by Saramagrean** [comment]
> > Yeah, should thank you HoaiTrung-san for hard work. I just edit cutscenes & made texture pack :)
> 
> sorry if my comment is not in the right place, I want to ask.? Are you the one who made the HD texture pack for Wariors Orochi 3, and can I request a texture font like this ![041ef00014d955f8beafed1b](https://user-images.githubusercontent.com/78854859/218578114-ba19c0db-d7bd-4a36-8369-0376720b1f06.png) ![041ef000436d254cbeafed1b](https://user-images.githubusercontent.com/78854859/218578121-3bcf988b-6a3e-4d21-988f-81e5e026f713.png) ![041ef00093e65e58beafed1b](https://user-images.githubusercontent.com/78854859/218578126-1a8943b3-1eec-4979-b366-654afd3b3b78.png) ![041ef000c8a7a294beafed1b](https://user-images.githubusercontent.com/78854859/218578133-fd7cb605-aa23-400a-84aa-a19d9143eabe.png) i want to download it on the ppsspp forum but, the size is too big for me, i only need 4 pictures can you share it.

Sure. :)

Wait a moment, I will upload soon.

**Comment by Saramagrean** [comment]
@IrfanH495 Here. [FONT.zip](https://github.com/Saramagrean/TKD_KWM-HD-Texture-Pack/files/11039992/FONT.zip)

**Comment by IrfanH495** [comment]
file Not Found
